Examples/Pose2SLAMG2O/main.swift

```diff
@@ -66,15 +66,15 @@ func main() {
   print("Initial error: \(problem.graph.error(val))")
   for _ in 0..<10 {
     let gfg = problem.graph.linearize(val)
-    let optimizer = CGLS(precision: 1e-6, max_iteration: 20)
+    let optimizer = CGLS(precision: 1e-6, max_iteration: 200)
```

Comment:
Is `20` not converging here?

Reply:
Yeah, when I run it on the INTEL dataset it seems to take 400ish iterations for the CGLS to converge to the `1e-6` precision. I previously put it at 20 because it was so slow and I wanted to see everything working without waiting forever.

Reply:
I see ;)